evasquez26/angular4-startup
# S6-ONE-234-239 Fixed error when refreshing the page on any route except home

## Story / Task

[ONE-234](https://bixlabs.atlassian.net/browse/ONE-234) and [ONE-239](https://bixlabs.atlassian.net/browse/ONE-239)


## What this PR does?

Fixes the error when refreshing the page on any route except the home page route '/'.


### Before 

- Whenever the user refreshed the page on a route different from the home page the app will throw an error of cannot GET X route, this only happened when using the server.js file

### After

- We can reload the page at any route and it will show the correct content

### How to test / reproduce

- Execute `USERNAME=<anyUserYouWant> PASSWORD=<anyPasswordYouWant> NODE_ENV=production API_URL=<theApiUrl> npm run start`
- Go to localhost:8080
- Enter the username and password you previously entered
- Enter the app and navigate to any route
- Refresh the page
- Observe that the page reloads correctly


### TODOs

N/A

### Related PRs

N/A

## Migrations

NO

Thanks!
